roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -125,9 +125,3 @@
 
 - name: Configure swap
   ansible.builtin.import_tasks: swap.yml
-
-- name: Set GID hugetlbfs user to match kolla pre-set
-  ansible.builtin.group:
-    name: hugetlbfs
-    gid: 42477
```

Comment:
I don't see this task is ensured elsewhere.
I would keep it, see https://review.opendev.org/c/openstack/tripleo-ansible/+/679809/8/tripleo_ansible/roles/tripleo-ovs-dpdk/tasks/workarounds.yml

Reply:
we still do this via https://github.com/openstack-k8s-operators/edpm-ansible/pull/334/files#diff-c9bbf4cad91988d4c06480b99a1cb929dc54992349898636da9d3920b32de34eR23

- { "name": "hugetlbfs", "gid": "42477", "group_only": true }

Reply:
Oh, sorry, the grep has failed me